### ISSUE TITLE
missing semicolon =)

### ISSUE THREAD

**Issue by jeneg** [8bitDesigner/ng-fastclick#5]
Hi! Tnx for wrapper, but please put missing semicolon after

```
angular.module('ng-fastclick', []).run(function() {
  FastClick.attach(document.body)
})
```

it breaks app after minification)

**Comment by 8bitDesigner** [comment]
Oh, you're totally right; thanks, mate!

**Comment by 8bitDesigner** [comment]
Fixed in [v1.0.2](https://github.com/8bitDesigner/ng-fastclick/releases/tag/v1.0.2)

**Comment by jeneg** [comment]
Great! Tnx you)
